qiskit/transpiler/passes/optimization/litinski_transformation.py

```diff
@@ -0,0 +1,44 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2025
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""Move clifford gates to the end of the circuit, changing rotation gates to multi-qubit rotations."""
+
+from qiskit.transpiler.basepasses import TransformationPass
+from qiskit._accelerate import litinski_transformation as litinski_transformation_rs
+
+
+class LitinskiTransformation(TransformationPass):
+    """
+    Applies Litinski transform to a circuit.
+
+    The transform applies to a circuit containing Clifford + RZ-rotation gates (including T and Tdg),
+    and moves Clifford gates to the end of the circuit, while changing rotation gates to multi-qubit
+    rotations.
+    """
+
+    def run(self, dag):
+        """Run the LitiskiTransformation pass on ``dag``.
+
+        Args:
+            dag (DAGCircuit): the input DAG.
+
+        Returns:
+            DAGCircuit: the output DAG.
+        """
+        new_dag = litinski_transformation_rs.run(dag)
+
+        # If the pass did not do anything, the result is None
+        if new_dag is None:
+            return dag
+        else:
+            return new_dag
```

Comment:
I'm surprised pylint did not complain about this 🤔 
```suggestion

        return new_dag
```

Reply:
Done in [bc76da4](https://github.com/Qiskit/qiskit/pull/14753/commits/bc76da40133040bc8ec01fd8d315dac8a030bdce).